avalon/tools/loader/model.py

```diff
@@ -1,17 +1,23 @@
 from ... import io, style
 from ...vendor.Qt import QtCore
-from ...vendor import qtawesome as qta
+from ...vendor import qtawesome
 
-from ..projectmanager.model import (
-    TreeModel,
-    Node
-)
+from ..models import TreeModel, Item
+from .. import lib as tools_lib
 
-from . import lib
+
+def is_filtering_recursible():
+    """Does Qt binding support recursive filtering for QSortFilterProxyModel?
+
+    (NOTE) Recursive filtering was introduced in Qt 5.10.
+
+    """
+    return hasattr(QtCore.QSortFilterProxyModel,
+                   "setRecursiveFilteringEnabled")
 
 
 class SubsetsModel(TreeModel):
-    COLUMNS = ["subset",
+    columns = ["subset",
```

Comment:
Could we make this `Columns`? It's to separate it from variables and methods.

Reply:
Sure. Fixed with [this commit](https://github.com/getavalon/core/pull/440/commits/8f798ca2699897b51fd9d0fe65eb158eed40a8f5).